### ISSUE TITLE
Retina icon

### ISSUE THREAD

**Issue by swrobel** [yegortimoshenko/paper#2]
It would be great to have a higher-resolution icon for those of us with retina displays. The included one is quite blurry. Is this addon still maintained?

**Comment by yegortimoshenko** [comment]
Yes, it is still maintained. It is quite stable (despite being v0.x) so I lack ideas on how to improve it. If you have any other suggestions, please write me back or open more issues.

I don't have a Retina display, so I've never thought about it before. All icons will need a redraw since their sources are low-resolution only. @bl will help me with these.

I'll ping you back when it's ready to upload to Mozilla Addons.

**Comment by Mno-hime** [comment]
I was wondering about the "success" icon: ![success](https://cloud.githubusercontent.com/assets/4632197/9032777/5a84d702-39c2-11e5-8ef5-76a1c625a9f4.png)

It's inconsistent with other icons, they rarely change visually that much when they try to inform user about something.

I'd suggest the icon ![dl](https://cloud.githubusercontent.com/assets/4632197/9032817/d25e4416-39c2-11e5-855c-f2d70d2909de.png) change to a blue (similarly to the Download icon): ![dl2](https://cloud.githubusercontent.com/assets/4632197/9032816/d25e2c7e-39c2-11e5-966d-1ee32dc295b3.png), or to green (=success in adding to Instapaper): ![dl4](https://cloud.githubusercontent.com/assets/4632197/9032937/8616a142-39c3-11e5-969c-6e0bf7e7b2b7.png). This icon should flip back to the default one after a second or so.
Failure should be visualized as a red icon: ![dl3](https://cloud.githubusercontent.com/assets/4632197/9032818/d2641454-39c2-11e5-8bb0-483f934747cf.png) This would be a new feature and should be accompanied with a notification: ![warning](https://cloud.githubusercontent.com/assets/4632197/9033141/03b6cac2-39c5-11e5-8eb4-c7cadd5f0cd9.png)

**Comment by yegortimoshenko** [comment]
I'll cut a new release without retina icons in a day, but after the release I will create retina icons as soon as I can and will release them as a part of `v1.1.0`. Sorry for inconvenience.
